src/test/java/pages/ElectronicaPage.java

```diff
@@ -0,0 +1,23 @@
+package pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+
+public class ElectronicaPage extends BasePage {
+    public ElectronicaPage(WebDriver driver) {
+        super(driver);
+    }
+
+    @FindBy(xpath = "//div[contains(text(),'Телевидение')]")
+    private WebElement tv_and_videoButton;
+    @FindBy(xpath = "//a[@href='https://catalog.onliner.by/tv']")
```

Comment:
the same story with the links
reason: if the application under test is localized we may face some problems with mapping